### DISCUSSION TITLE
Auto-complete not working with 1.69.0-nightly on windows 11

### DISCUSSION THREAD

**Discussion by ChaseLewis** [rust-lang/rust-analyzer#14194] [Q&A]
If I install stable everythings great, but I was wanting to try some nightly features and I lose auto-complete and type hints even after reinstalling rust-analyzer. Any ideas what I can do.

IDE: VSCode
OS: Windows 11
RUST: 1.69.0-nightly

rustc --version correctly reports 1.69.0-nightly so not sure what is the issue.

If I save the file or build I get updated error squiggles, but nothing without a file change.

**Comment by lnicola** [comment]
Is that one specific nightly? I tested on Linux, on `rustc 1.69.0-nightly (fdbc4329c 2023-02-22)`, and it seems to be working fine.

Do you have anything in the logs?

No errors before saving is a different issue, haven't looked into it yet, but I can confirm it.
